### PR TITLE
add a3 sample

### PR DESCRIPTION
**Pull request by stevenBorisko**

### CODE REVIEW

File: samples/mig-with-container/a3_multi_nic_network/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+1. [nvidia-h100-80gb](https://cloud.google.com/compute/docs/gpus) GPU.
+1. [Multi-NIC VPC](https://cloud.google.com/vpc/docs/create-use-multiple-interfaces) in the project.
+1. [NFS Filestore mounted in the container](https://cloud.google.com/filestore)
+1. GCS Bucket mounted in the container via [GCSFuse](https://cloud.google.com/storage/docs/gcs-fuse).
+1. [COS-Cloud machine image](https://cloud.google.com/container-optimized-os/docs).
+1. [Deep Learning Container image](https://cloud.google.com/deep-learning-containers).
+   to be executed when the VM boots up.
+
+## Usage via Docker Image
+Please find detailed set up instruction for docker image
```
Comment (Chris113113) [inline samples/mig-with-container/a3_multi_nic_network/README.md:15]: Tell, don't ask

File: samples/mig-with-container/a3_multi_nic_network/main.tf [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+variable "disk_size_gb" {}
+variable "disk_type" {}
+variable "filestore_new" {}
+variable "gcsfuse_existing" {}
+variable "labels" {}
+variable "machine_type" {}
+variable "network_config" {}
+
+module "a3-mig" {
+  source = "github.com/GoogleCloudPlatform/ai-infra-cluster-provisioning//terraform/modules/cluster/mig"
```
Comment (Chris113113) [inline samples/mig-with-container/a3_multi_nic_network/main.tf:16]: `provisioning//terraform` intentional?

File: samples/mig-with-container/a3_multi_nic_network/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+[here](../../../README.md#usage-via-docker-image)
+
+Please copy the [terraform.tfvars](./terraform.tfvars) file to your current working
+directory of your local machine. Then follow the below instructions to create the cluster.
```
Comment (Chris113113) [inline samples/mig-with-container/a3_multi_nic_network/README.md:19]: Add line about which tfvar values need to be set, i.e. project_id, remote_mount

**Review by Chris113113 — CHANGES_REQUESTED** [review]
Tell don't ask

File: samples/mig-with-container/a3_multi_nic_network/main.tf (hunk 2 above)
Comment (stevenBorisko) [reply samples/mig-with-container/a3_multi_nic_network/main.tf]: unfortunately, it is

**Review by sdlin — APPROVED** [review]

**Review by Chris113113 — APPROVED** [review]

**Pull request merged**